projects/dsp-ui/src/lib/viewer/services/value-operation-event.service.ts

```diff
@@ -1,6 +1,7 @@
 import { ReadValue } from '@dasch-swiss/dsp-js';
 import { Subject, Subscription } from 'rxjs';
 import { filter, map } from 'rxjs/operators';
+import { BaseValue } from '@dasch-swiss/dsp-js/src/models/v2/resources/values/base-value';
```

Comment:
wrong import path, it should only be "@dasch-swiss/dsp-js"

Reply:
<img width="725" alt="Screen Shot 2020-08-11 at 16 17 19" src="https://user-images.githubusercontent.com/60604010/89909281-315bfc00-dbef-11ea-9d1f-4deeaa1f0f44.png">

I think I may need to add the BaseValue to the DSP-JS exports?

Reply:
yes you have to export it

Reply:
done in 68cbdf9